ttnn/cpp/ttnn/operations/matmul/device/matmul_op.cpp

```diff
@@ -1045,16 +1045,16 @@ Tensor matmul(
     const std::optional<Tensor>& optional_output_tensor) {
     std::vector<std::optional<const Tensor>> optional_input_tensors = {};
     std::vector<Tensor> output_tensors;
-    const bool opt_output_empty = !optional_output_tensor.has_value();
+    const bool is_optional_output_tensor_empty = !optional_output_tensor.has_value();
```

Comment:
Should change.
This implies that the optional output tensor does not have values.

Maybe change to is_optional_output_tensor

Reply:
flag is looking if its empty. Based on this flag, code will populate output_tensors

Reply:
I wanna be verbose & add "empty" to the name. Will edit it to ur name